src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -324,6 +326,29 @@ public Response validateOtpKeyForService(@Parameter(schema = @Schema(implementat
                         .orElseGet(() -> Response.status(OK).build()));
     }
 
+    @POST
+    @Path(INVITES_RESOURCE_V2 + "/otp/validate")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    @Operation(
+            summary = "Validates OTP for the invite - part of creating service",
```

Comment:
Can remove "- part of creating service" from this summary

Reply:
Good spot!